test/integration/apiserver/apply/apply_test.go

```diff
@@ -106,6 +106,7 @@ func TestApplyAlsoCreates(t *testing.T) {
 			Namespace("default").
 			Resource(tc.resource).
 			Name(tc.name).
+			Param("fieldManager", "apply").
 			Body([]byte(tc.body)).
```

Comment:
Future work: we should make shortcut functions that call apply in client-go.

Reply:
Completely agree. I'm also talking to kubebuilder folks about that.

Reply:
Let me add this to the list of tasks that we have in the umbrella.